contributor/de/04-mechanics-of-contributing-article-de.asciidoc

```diff
@@ -0,0 +1,158 @@
+Wenn Sie die Menschen sprechen hören können und ihre Eigenheiten kennen, wird die Zusammenarbeit aus der Ferne einfacher.
+
+==== Vermeidung von Ablehnung
+
+Haben Sie eine große Funktion, die Sie beisteuern möchten?
+Ausgezeichnet!
+Wäre es nicht furchtbar, wenn Ihre ganze Arbeit umsonst wäre?
+Das kann passieren, wenn das Team des Gastgebers bereits etwas sehr Ähnliches entwickelt, die Software veraltet ist oder das, was Sie vorschlagen, nicht in sein Projekt passt.
+Dieses Problem tritt häufig auf, und viele Teambeziehungen haben darunter gelitten, dass man sich nicht im Voraus einig war, dass ein Beitrag gut passt.
+
+Machen Sie sich selbst und das Team des Gastgebers glücklich (und sparen Sie möglicherweise etwas Arbeit), indem Sie die Zustimmung des Teams des Gastgebers zum Benutzer/technischen Design des Beitrags einholen, _bevor_ Sie an den Änderungen arbeiten und einen Pull Request einreichen.
+Sie müssen verstehen, wie das Team des Gastgebers möchte, dass Sie dies erreichen.
+Am besten fragen Sie einen https://innersourcecommons.org/de/learn/learning-path/trusted-committer[_Trusted Committer_], wie Sie Ihren Vorschlag am besten besprechen können.
+
+Es ist eine immer wieder bewiesene Weisheit aus dem Open Source Bereich, dass, wenn Sie wählen können, wie Sie Ihren Vorschlag diskutieren wollen, Sie versuchen sollten, einen schriftlichen Weg zu wählen.
+Idealerweise wählen Sie den Weg, bei dem die Artefakte öffentlich, durchsuchbar und dauerhaft verlinkbar sind, um es zu ermöglichen, in späteren Diskussionen über diesen zukünftigen Beitrag oder andere Beiträge - von Ihnen oder anderen - auf Ihren Vorschlag zu verweisen.
+
+Diese Art von frühzeitiger Einigung auf hohem Niveau spart Zeit bei der Überarbeitung oder Ablehnung Ihres Pull Requests im Nachhinein.
+
+=== Erstellen des Pull Requests
+
+==== Kommunikation und Entblockierung
+
+Großartig, Sie haben sich mit der Herangehensweise des Host-Teams vertraut gemacht und sie freuen sich darauf, Ihren Pull Request zu erhalten.
+Welche Stolpersteine warten nun auf Sie?
+
+Erstens werden Sie in weniger direktem Kontakt mit ihnen stehen.  Zweitens wird von Ihnen nicht erwartet, dass Sie so sachkundig und kompetent sind, wie Sie es vielleicht bei den Vollzeitprojekten Ihres Teams sind.
+Wie können Sie nun am besten damit umgehen?
+
+Versuchen Sie, die Dokumentation, die Gesprächsarchive und die Code-Artefakte des Gastteams durchzusehen, um sich selbst zu entlasten.
+Dies ist ähnlich wie die Situation, in der Sie und wahrscheinlich die meisten Leute sich befinden, wenn sie eines der beliebten OSS-Projekte verwenden.
+
+Ähnlich wie bei Open-Source-Projekten sollten Sie das Host-Team fragen, wenn die Dinge nicht vorankommen, selbst wenn Sie versuchen, die Blockade zu lösen.
+Die Fragen, die Sie stellen, und die Antworten, die Sie erhalten, werden anderen helfen, die nach Ihnen kommen, die gleichen Probleme zu lösen.
```

Comment:
```suggestion
Die Fragen, die Sie stellen, und die Antworten, die Sie erhalten, werden anderen helfen, die nach Ihnen kommen und die die gleichen oder ähnliche Fragen stellen ihre Probleme zu lösen.
```
Technically that's correct German. It might be warranted though to split it into two sentences or simplify the language.